rust_book_ex_8_3/src/string_ex.rs

```diff
@@ -0,0 +1,21 @@
+fn starts_with_vowel(s: &String) -> bool {
+    let vowels = vec!['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
```

Comment:
Ouch, new vector allocation every time you call this, this is evil. (see example below, uses static constant string)